.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 vendor
-
+/composer.lock
+.idea
```

Comment:
Please remove the `.idea` as it should be set in your global gitignore instead of per project when you use this editor

Reply:
Removed.  Also added another null-safe fix to JobInformation model